Dockerfile

```diff
@@ -1,15 +1,26 @@
-FROM concourse/busyboxplus:git
+FROM golang:1.5.1-alpine
```

Comment:
Again, can we use Go 1.5.3 here for the security fix - https://groups.google.com/d/topic/golang-announce/MEATuOi_ei4/discussion